addons/web_editor/models/ir_qweb.py

```diff
@@ -60,6 +75,12 @@ def _compile_directive_snippet(self, el, options):
         )
         return [self._append(ast.Str(div))] + self._compile_node(el, options) + [self._append(ast.Str(u'</div>'))]
 
+    def _compile_directive_snippet_call(self, el, options):
+        el.set('t-call', el.attrib.pop('t-snippet-call'))
+        key = el.attrib.get('t-call')
```

Comment:
Instead of pop -> set -> get, can't you switch those two lines to have pop (in key) -> set ?